actionview/lib/action_view/helpers/rendering_helper.rb

```diff
@@ -8,21 +8,17 @@ module Helpers
     module RenderingHelper
       # Returns the result of a render that's dictated by the options hash. The primary options are:
       #
-      # * <tt>:partial</tt> - See <tt>ActionView::PartialRenderer</tt>.
-      # * <tt>:file</tt> - Renders an explicit template file (this used to be the old default), add :locals to pass in those.
-      # * <tt>:inline</tt> - Renders an inline template similar to how it's done in the controller.
-      # * <tt>:text</tt> - Renders the text passed in out.
-      # * <tt>:plain</tt> - Renders the text passed in out. Setting the content
-      #   type as <tt>text/plain</tt>.
-      # * <tt>:html</tt> - Renders the HTML safe string passed in out, otherwise
-      #   performs HTML escape on the string first. Setting the content type as
-      #   <tt>text/html</tt>.
-      # * <tt>:body</tt> - Renders the text passed in, and inherits the content
-      #   type of <tt>text/plain</tt> from <tt>ActionDispatch::Response</tt>
-      #   object.
+      # * <tt>:partial</tt> - See <tt>ActionView::PartialRenderer</tt> for details.
+      # * <tt>:file</tt> - Renders an explicit template file. Add <tt>:locals</tt> to pass in, if so desired.
+      # * <tt>:inline</tt> - Renders an inline template.
+      # * <tt>:text</tt> - Renders provided text.
```

Comment:
I've been poking around to determine what documentation should be added in the appropriate places but I don't yet have clarity on how to piece the suggestions together so that these edits are accurate. But I'm still researching... 🔍 
_(This has been a great way to be introduced to the code base - having something specific to work toward. Thanks again for your input!)_